app/views/app/index.html.haml

```diff
@@ -1,2 +1,4 @@
-= react_component 'ScreensmartApp', {csrfToken: form_authenticity_token},
-                                    class: 'app'
+:javascript
+  window.csrfToken = '#{form_authenticity_token}'
```

Comment:
Maar waarom willen we van jquery af? Als het puur de grootte is, dan heb je ook zepto.js of jquip.js, waarbij de interface hetzelfde is.

Ik zie gewoon niet waarom iemand ooit ajax zonder promises interface zou wil gebruiken. 
Of events wil afhandelen zonder makkelijke proxy.